admin/hardware-inventory.md

```diff
@@ -4,9 +4,9 @@
 
 | Item | Units | Status |
 |:-----|:------|:-------|
-| Raspberry Pi 3B+ with power | 6 | With @benhylau |
-| Toplinkst TOP-S5 | 45 | Waiting for USB adapters |
-| Samsung Evo Select 32 GB | 20 | With @benhylau |
+| Raspberry Pi 3B+ with power | 6 | With @darkdrgn2k |
+| Toplinkst TOP-S5 | 45 | With @darkdrgn2k |
```

Comment:
```suggestion
| Toplinkst TOP-S5 | 40 | With @darkdrgn2k |
```

Reply:
Sorry I forgot to subtract this. It's `50-(3+2+3+1+1) = 40`, every single one is accounted for in the _Hardware allocation_ table.

Thanks for noticing this @dasanchez